README.md

```diff
@@ -36,27 +36,27 @@ Windows 10 and above.
 
 _komorebi_ allows you to control application windows, virtual workspaces and display monitors with a CLI which can be
 used with third-party software such as [`whkd`](https://github.com/LGUG2Z/whkd)
-and [AutoHotKey](https://github.com/Lexikos/AutoHotkey_L) to set user-defined keyboard shortcuts.
+and [AutoHotKey](https://github.com/AutoHotkey/AutoHotkey) to set user-defined keyboard shortcuts.
 
 _komorebi_ aims to make _as few modifications as possible_ to the operating
 system and desktop environment by default. Users are free to make such
 modifications in their own configuration files for _komorebi_, but these will
 remain opt-in and off-by-default for the foreseeable future.
 
 Please refer to the [documentation](https://lgug2z.github.io/komorebi) for instructions on how
-to [install](https://lgug2z.github.io/komorebi/installation.html) and
-[configure](https://lgug2z.github.io/komorebi/example-configurations.html)
-_komorebi_, [common workflows](https://lgug2z.github.io/komorebi/common-workflows/komorebi-config-home.html), a complete
+to [install](https://lgug2z.github.io/komorebi/installation) and
```

Comment:
These are also fine to keep with the .html extension since this is how the mkdocs site generates the pages